website/docs/addon_usd_artist_get_started.md

```diff
@@ -0,0 +1,98 @@
+---
+id: addon_usd_artist_get_started
+title: Getting Started
+sidebar_label: Getting Started
+---
+
+:::info
+This page is still under construction 👷🚧🛠️🔜🏗️.
+:::
+
+# Getting started
+
+## Creating an asset
+
+The producer has his 🔫 pointing at you and needs the **asset** _now!_
+
+When we are referring to the USD asset, we mean the asset with both the model's geometry and the look's materials and textures. The USD structure (simplified) then becomes:
+
+```
+asset.usd
+  - look.usd
+  - model.usd
+```
+Where the look is applied over the model inside the asset.
+
+#### Creating the model
```

Comment:
I am lacking some ```publish instance``` settings and media for such process as its quite important (from what I can remember from demo session) publishing options need some cover here.